crates/router/src/services/authentication.rs

```diff
@@ -1033,6 +1033,7 @@ where
     }
 }
 
+/*
 pub struct JWTAuthMerchantOrProfileFromRoute {
```

Comment:
Can we remove this, or is there a reason why this is kept?

Reply:
I'll address the changes around the webhook events list endpoints in a separate PR, will revisit it then. I've intentionally included changes related to the webhook events list endpoints in one commit, so that I can revisit changes from that single commit.

Reply:
Not mandatory, but if you are making any changes in this PR, can you add a TODO comment.